app/displayers.py

```diff
@@ -0,0 +1,16 @@
+from app.book import Book
+
+
+class Displayer:
+    def display(self, book: Book) -> None:
+        raise NotImplementedError
```

Comment:
In my opinion it's better to use abstract classes & abstract methods cause the use of abstract classes can provide more flexibility and clarity in defining the class hierarchy.